components/Header.tsx

```diff
@@ -0,0 +1,38 @@
+import Link from "next/link";
+import { FC, PropsWithChildren } from "react";
+
+
+const Header: FC<
+  PropsWithChildren<
+    any
```

Comment:
I think you can use `never` instead of `any` here. The difference between the two is that with `any` this component would accept any props that were passed, while with `never` typescript checker will complain with any other props besides `children` 😃